tests/h/services/search_index/_queue_test.py

```diff
@@ -13,14 +13,16 @@
 
 LIMIT = 2
 ONE_WEEK = datetime_.timedelta(weeks=1)
+ONE_WEEK_IN_SECONDS = int(ONE_WEEK.total_seconds())
 MINUS_FIVE_MINUTES = datetime_.timedelta(minutes=-5)
+MINUS_FIVE_MINUTES_IN_SECONDS = int(MINUS_FIVE_MINUTES.total_seconds())
```

Comment:
This is causing a few lanky lines. I think we could get away with `MINUS_5_MIN` `MINUS_5_MIN_IN_SEC` etc. Might pull them back under? Bit gross, but hey

Reply:
Changed it to `MINUS_5_MIN_IN_SECS`